tech-stacks-dc.md

```diff
@@ -25,72 +25,44 @@ Once everyone has researched their assigned technologies your group will come to
 You will need to create three presentation slides using Google Slides (details will be provided by your instructor). On your three slides you should present an overview of the tech stack as a whole and also dive into the major technologies being used.
 
 ## Example Tech Stacks
-Below we present four different tech stacks for real-world applications. What we're presenting below is somewhere between the reality of the projects being hosted and an outside observer's hypothesis of what stack they are using. While each of these projects has a core piece that is open source most of them involve some proprietary software and/or hosting, which makes it difficult to say for certain what technologies are being used for that part of the stack.
+Below we present four different tech stacks for previous Ada capstone projects. The technology choices listed here are based on the available code from those projects and may not reflect the complete set of technologies that were used. If your assigned tech stack has fewer technologies you should take the opportunity to investigate each technology in more detail. We've selected two mobile applications and two web applications:
```

Comment:
Can you add something about comparing/contrasting to their experience in Ruby/Rails/JS?